src/modules/Reviews/Review.ts

```diff
@@ -0,0 +1,6 @@
+// src/entities/Review.ts
+class Review {
+  constructor(public id: string, public userId: string, public text: string) {}
```

Comment:
I think this should use private properties

Reply:
Revised to private variables, thanks.